runner.go

```diff
@@ -51,6 +52,13 @@ func (r *runner) startRun(c cliContext) error {
 		return monitorRunStatus(c, runID)
 	}
 
+	// Validate that --fail-fast and --max-retries are not used together
+	failFast := c.Bool("fail-fast")
+	maxRetries := c.Uint("max-retries")
+	if failFast && (maxRetries > 0) {
```

Comment:
Hmm, this check does mean that the following invocation is invalid:
```
$ rainforest-cli --skip-update run --max-retries 1 --fail-fast 123
2020/12/17 15:43:34 The --fail-fast and --max-retries flags cannot be used together
```

but this is fine:
```
$ rainforest-cli --skip-update run --max-retries 1 --fail-fast 123
2020/12/17 15:43:54 RF API Error (404): Cannot find record
```

which is a bit weird?

Reply:
Wait, what's the difference? Did you mean to do `--max-retries 0` in the second one?

Reply:
Yep - bad copy/paste.

Reply:
```
$ ./rainforest-cli --skip-update run --max-retries 0 --fail-fast 123
2020/12/18 10:15:54 RF API Error (404): Cannot find record
$
```

Reply:
Right, so what's weird about that? `max-retries` defaults to 0, so if you set it to `0` it's the same as not setting it, since there will be no reruns - so you can do `fail-fast`.

Reply:
It's weird because the error message says that `The --fail-fast and --max-retries flags cannot be used together` but it turns out they can - so ideally the error message should reflect this or the conditional should be "did the user set the fail-fast and max-retries argument" - I don't know if the library we're using lets us determine that though.

It's not worth sweating about too much.

Reply:
Updated the error message to
```
You can't use --fail-fast when --max-retries is greater than 0. For the CLI to retry on failure, it has to wait until completion.
```